compiler/fir/tree/gen/org/jetbrains/kotlin/fir/declarations/FirProperty.kt

```diff
@@ -45,6 +45,7 @@ abstract class FirProperty : FirVariable<FirProperty>(), FirTypeParametersOwner,
     abstract override val symbol: FirPropertySymbol
     abstract val backingFieldSymbol: FirBackingFieldSymbol
     abstract val isLocal: Boolean
+    abstract val isFromVararg: Boolean
```

Comment:
From my point-of-view being originated from a vararg parameter is not a substantial feature of a property.
It's really expected to be used and initialized just once while all the implementations should override it and all users will see it in completion.

For such kind of smaller features, you may use `org.jetbrains.kotlin.fir.declarations.FirDeclarationDataRegistry` and `org.jetbrains.kotlin.fir.declarations.FirDeclarationAttributes`

Reply:
Thanks for the suggestion! I didn't know about FirDeclarationAttributes. It's very handy for the boolean value I need.